applications/DEMApplication/custom_constitutive/DEM_D_Linear_Simple_Coulomb_CL.h

```diff
@@ -14,22 +14,22 @@
 #pragma once
 
 #include <string>
+#include "../custom_elements/spheric_particle.h"
 #include "DEM_discontinuum_constitutive_law.h"
 
 namespace Kratos {
 
-  class SphericParticle;
```

Comment:
Out of curiosity, why do you need to remove this for this PR?

Reply:
It was crashing in my machine. Also in the CI.

Reply:
Ok, but does anyone understand why? I just don't see how the quadratures are linked with the DEM application...

Reply:
Maybe @roigcarlo can shed some light but forward declarations are evil...